pom.xml

```diff
@@ -0,0 +1,184 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright © 2020 Cask Data, Inc.
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License"); you may not
+  ~ use this file except in compliance with the License. You may obtain a copy of
+  ~ the License at
+  ~
+  ~ http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+  ~ WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+  ~ License for the specific language governing permissions and limitations under
+  ~ the License.
+  -->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>io.cdap.plugin</groupId>
+  <artifactId>switch</artifactId>
+  <version>1.0.0-SNAPSHOT</version>
+  <packaging>jar</packaging>
+
+  <name>Switch</name>
+  <description>
+    A collection of plugins that allow you to specify switch..case type logic in CDAP
+  </description>
+
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <cdap.version>6.1.2</cdap.version>
+    <cdap.plugin.version>2.2.0</cdap.plugin.version>
+    <!-- This is the version range for the app.parents that this plugin is valid for. Usually this will correspond with
+         the CDAP version, but not always. -->
+    <etl.versionRange>[6.1.0-SNAPSHOT,7.0.0-SNAPSHOT)</etl.versionRange>
+    <!-- This is the list of packages where your code exists. You can add multiple and separated with a ; or use wildcards *. -->
+    <exported-packages>io.cdap.plugin.*</exported-packages>
```

Comment:
should be more specific, like io.cdap.plugin.switch.* to avoid accidental overlap of exports

Reply:
yup, forgot to update after moving class to the switch sub-package. Fixed.